src/app/src/main/java/com/example/bookmark/abstracts/AddEditBookActivity.java

```diff
@@ -16,6 +16,9 @@
 import com.example.bookmark.fragments.ImageSelectDialogFragment;
 import com.google.android.material.textfield.TextInputLayout;
 
+/**
+ * @author Eric Claerhout
+ */
```

Comment:
@imswebra Make sure to add a high-level Javadoc description to all the classes you author.